server_addon/houdini/server/settings/create.py

```diff
@@ -17,6 +18,58 @@ class CreateArnoldAssModel(BaseSettingsModel):
         default_factory=list,
     )
     ext: str = SettingsField(Title="Extension")
+    staging_dir: str = SettingsField(title="Staging Directory")
+
+
+class CreateArnoldRopModel(BaseSettingsModel):
+    enabled: bool = SettingsField(title="Enabled")
+    default_variants: list[str] = SettingsField(
+        title="Default Products",
+        default_factory=list,
+    )
+    render_staging_dir: str = SettingsField(title="Render Staging Directory")
+    ass_dir: str = SettingsField(title="Ass Directory")
+
+
+class CreateKarmaROPModel(BaseSettingsModel):
+    enabled: bool = SettingsField(title="Enabled")
+    default_variants: list[str] = SettingsField(
+        title="Default Products",
+        default_factory=list,
+    )
+    render_staging_dir: str = SettingsField(title="Render Staging Directory")
+    checkpoint_dir: str = SettingsField(title="Checkpoint Directory")
+    usd_dir: str = SettingsField(title="USD Directory")
```

Comment:
Would it be more generalizable if we find a common terminology/name for a Render Scene Description or File so that we don't need unique keys for each renderer but can have just something that describes "exported render scene description" or whatever. Of course the titles can be more specific. Just wonder whether we want to consolidate the logic more for potential re-use?

Reply:
For these creators we can use the same settings model if we picked up an name for the intermediate exported files.
- `CreateVrayROPModel`
- `CreateRedshiftROPModel`
- `CreateMantraROPModel`
- `CreateArnoldRopModel`

Unlike Karma, it has more options 
- `CreateKarmaROPModel`

---
What could be a good name ? 
- proxy scene
- scene description
- scene definition
- exported file 

I think we can create an issue for it so we can change that name along the plugins to ensure consistency.. what do you think ?